core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/chain/StopStartUpscaleFlowEventChainFactory.java

```diff
@@ -56,6 +59,7 @@ private void addStackSyncTriggerEvent(StackAndClusterUpscaleTriggerEvent event,
     private void addClusterScaleTriggerEventIfNeeded(StackAndClusterUpscaleTriggerEvent event, StackView stackView, ClusterView clusterView,
```

Comment:
the ClusterView does not use anymore in the function. Could you please remove it? And ClusterView does not need to fetch any more

Reply:
Done